plugindocs.rb

```diff
@@ -130,6 +132,27 @@ def tag(version)
     version ? "v#{version}" : "main"
   end
 
+  # returns alias mappings for each plugin type & name ([type][target]=alias) ex: (["input"]["beats"]="agent")
+  def load_alias_mappings
+    alias_util = Util::AliasPluginUtil.new
+    yaml = alias_util.fetch_alias_mappings
```

Comment:
Can we pull this index inversion up into `Util::AliasPluginUtil` as something like `Util::AliasPluginUtil#fetch_reverse_index`?

Additionally, the `yaml` variable doesn't contain _YAML_ -- it contains an object that was deserialized from _YAML_, but that is a detail that this scope doesn't need to care about.

Reply:
Overall, simplified by removing repetitive logics.